public/javascripts/data-cart.js

```diff
@@ -67,7 +67,7 @@ const cartData = [
                     price: 350,
                     formatted_price: '350',
                     total_amount: 1050,
-                    formatted_total_amount: '1.050',
+                    formatted_total_amount: '1,050',
```

Comment:
for future reference we can make function to make this dynamic

Reply:
Definitely